grails-app/services/au/org/ala/ecodata/SpatialService.groovy

```diff
@@ -417,6 +417,7 @@ class SpatialService {
                                 }
 
                                 intersectedObjects.sort { it.name }
+                                println obj.name + intersectedObjects
```

Comment:
is this required?

Reply:
oops, that came along for the ride with some testing I was doing early. I've deleted it now.